staging/publishing/rules.yaml

```diff
@@ -0,0 +1,1307 @@
+skip-source-branches:
+- release-1.5
+- release-1.6
+recursive-delete-patterns:
+- BUILD
+- "*/BUILD"
+- BUILD.bazel
+- "*/BUILD.bazel"
+- Gopkg.toml
+rules:
+- destination: code-generator
+  branches:
+  - source:
+      branch: master
+      dir: staging/src/k8s.io/code-generator
+    name: master
+  # - source:
+  #     branch: release-1.8
```

Comment:
@sttts can we remove these or do you want to keep these commented? IMO we can add these back if we ever need to publish them again (super super critical security fix). Because right now this yaml file is huge. :)

Reply:
Absolutely non-critical, but maybe remove them in a separate commit (even in the same PR) so we have them in the git history of `k/k`?

Reply:
@hoegaarden good idea.

Btw, judging from the last apimachinery CVE, we don't backport that far anyway. @liggitt what's the rule?

Reply:
also note, we have: `skip-source-branches:` to skip old branches. That's easier to use than commenting tons of yaml lines.

Reply:
remove everything up to 1.9.